java/lang/String/StringTest.cpp

```diff
@@ -307,7 +307,7 @@ TEST (JavaLang, StringCopyValueOf) {
     }
 }
 
-    // FIXME
+// FIXME

```

Comment:
Please fix this or describe why you cant fix.

Reply:
Yep, I just reformat code style, maybe you should ask the author first.